src/components/passphrase/steps.js

```diff
@@ -0,0 +1,45 @@
+export default context => ({
+  info: {
+    cancelButton: {
+      title: 'cancel',
+      onClick: () => { context.props.closeDialog(); },
+    },
+    confirmButton: {
+      title: 'next',
+      onClick: () => { context.setState({ currentStep: 'generate' }); },
+    },
+  },
+  generate: {
+    cancelButton: {
+      title: 'cancel',
+      onClick: () => { context.props.closeDialog(); },
+    },
+    confirmButton: {
```

Comment:
Yes I knew about that. This is an issue with Dialog component. There are also some other improvements required on that component. I'll create a ticket for it